clap_mangen/src/render.rs

```diff
@@ -220,7 +220,12 @@ pub(crate) fn subcommands(roff: &mut Roff, cmd: &clap::Command, section: &str) {
     for sub in cmd.get_subcommands().filter(|s| !s.is_hide_set()) {
         roff.control("TP", []);
 
-        let name = format!("{}-{}({})", cmd.get_name(), sub.get_name(), section);
+        let name = format!(
+            "{}-{}({})",
+            cmd.get_display_name().unwrap_or_else(|| cmd.get_name()),
+            sub.get_name(),
+            section
+        );
         roff.text([roman(&name)]);
```

Comment:
Apparently, we have no test cases that show use of subcommands?

Could you add a test that covers this?

Reply:
@epage Added a test case.